OneSignalSDK/onesignal/core/src/main/java/com/onesignal/user/IUserManager.kt

```diff
@@ -1,6 +1,7 @@
 package com.onesignal.user
 
 import com.onesignal.OneSignal
+import com.onesignal.user.internal.state.IUserStateObserver
```

Comment:
since `IUserManager` is part of the external interface, it should not reference anything under an `internal` package.  I recommend you move `IUserStateObserver`, `UserChangedState`, and `UserState` under the `com.onesignal.user` package as they will also be part of the external API.